zerver/lib/outgoing_webhook.py

```diff
@@ -187,7 +187,7 @@ def notify_bot_owner(event: Dict[str, Any],
     send_response_message(bot_id, message_info, notification_message)
 
 def request_retry(event: Dict[str, Any],
-                  request_data: Dict[str, Any],
+                  request_data: Optional[Dict[str, Any]],
```

Comment:
This is the wrong fix for this one.  `do_rest_call` actually shouldn't accept `None` for `request_data`; the tests that pass None are buggy (and should pass `{}` or something instead).

Reply:
I've dropped this for now; this file ended up being in the excluded list in any case.